src/main/java/org/jitsi/jigasi/SipGatewaySession.java

```diff
@@ -509,12 +509,30 @@ public void outgoingCallCreated(CallEvent callEvent)
                     if(roomName != null)
                     {
                         Call call = callEvent.getSourceCall();
-                        call.setData("EXTRA_HEADER_NAME.1",
+                        AtomicInteger headerCount = new AtomicInteger(0);
```

Comment:
It doesn't hurt to use an atomic int but I'm struggling to understand what's its purpose this particular case 🤔

Reply:
This is because in the lambda the variables need to be final or effectively final... And this int is incremented and used there.

Reply:
got it, thanks damencho